numpydoc/docscrape.py

```diff
@@ -327,6 +327,11 @@ def _parse(self):
             if not section.startswith('..'):
                 section = (s.capitalize() for s in section.split(' '))
                 section = ' '.join(section)
+                if self[section]:
```

Comment:
you dont want keyerror here for unknown sections

14.9.2016 0.07 "Nelle Varoquaux" <notifications@github.com> kirjoitti:

> In numpydoc/docscrape.py
> <https://github.com/numpy/numpydoc/pull/65#discussion_r78653691>:
>
> > @@ -327,6 +327,11 @@ def _parse(self):
> >              if not section.startswith('..'):
> >                  section = (s.capitalize() for s in section.split(' '))
> >                  section = ' '.join(section)
> > +                if self[section]:
>
> Why would that be better? If section is not a key of the dictionary (nor
> starts with ..), an error will be raise later on (l.344). But maybe I am
> missing something.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/numpy/numpydoc/pull/65/files/c3e60b86bf6e493649d476ad5d5a60b557afaa02#r78653691>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACI5h84FYPNrWGYb-dMfi3ciFvrIIpsks5qpx59gaJpZM4J76JG>
> .
>